src/templates/login/login.tsx

```diff
@@ -40,6 +45,22 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         defaultMessage: "inloggen",
       });
 
+  const _labelOidcLogin = labelOidcLogin
+    ? labelOidcLogin
+    : intl.formatMessage({
+        id: "mykn.templates.Login.labelOidcLogin",
+        description: "templates.Login: The label for the OIDC login button.",
+        defaultMessage: "OIDC Login",
```

Comment:
Maybe improve the (Dutch) default label. I prefer something short but at least somewhat B1, e.g. ("Inloggen met organisatie account"). 

Don't forget to create the translations files:

> Messages can be extracted by running npm run makemessages after which they end up in src/lib/i18n/messages, they can be compiled after translation using npm run compilemessages.